### DISCUSSION TITLE
[Feature] Add thumbnails of albums im picture details on mobile App

### DISCUSSION THREAD

**Discussion by Frava735** [immich-app/immich#11615] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Mobile app version 1.111.x

When clicking on the 'i' for any picture, it shows a lot of information, but never the album thumbnail it is present in.
Multiple album thumbnails should be displayed when picture is part of multiple albums (same goes for the faces... multiple faces, multiple thumbnails).

These album thumbnails would be a link to the album it represents.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by AFumi39** [comment]
I would like this feature too. Now is very uncomfortable, since you have to switch between Android App and Web interface according to the function you need to use (just for example, you have to use the App to rotate a Photo, then use the Web to view the albums it belongs to).

Hope this function will be added soon 🤞

**Comment by Victor-Hugenay** [comment]
I would appreciate to have this feature 🙌🏼
